department-of-veterans-affairs/abd-vro
# Feature/mcp 2011-JWT-Validation

## What was the problem?
Add JWT token validation for MAS endpoints 

Associated tickets or Slack threads:

- [MCP-2011](https://amida.atlassian.net/browse/MCP-2011)

## How does this fix it?[^1]
The JWT in the HTTP header is used to validate MAS endpoints.

## How to test this PR
- Step 1
  Add a valid JWT token  as a Bearer token 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
